.github/workflows/test-projects.yml

```diff
@@ -39,6 +38,11 @@ jobs:
               name: "oban",
               test-cmd: "mix test --only igniter",
             },
+            {
+              org: "mishka-group",
+              name: "mishka_chelekom",
+              test-cmd: "mix test",
```

Comment:
I'd prefer to only run tests related to the igniter installers, can you add a `---only igniter` here and tag the relevant tests? Even if they are all igniter tests at the moment, the point is to make sure igniter's build doesn't fail just because some unrelated test fails in mishka.

Reply:
@zachdaniel Yes, I understand. now my master branch tests have igniter flag and i edited the `.github/workflows/test-projects.yml` and added `mix test --only igniter`
Thank you